test/test_utils.h

```diff
@@ -39,18 +39,18 @@ struct Resolution {
 
     inline bool operator <=(const Resolution& other) const
     {
-        return (width <= other.width) and (height <= other.height);
+        return (width <= other.width) && (height <= other.height);
```

Comment:
these changes just because VS does not support iso646?

Reply:
Yes, the ciso646.h header will be deprecated too so I thought it was easier to use the non-iso646 symbols